_layouts/ogc-flavor.adoc

```diff
@@ -20,8 +20,10 @@ ogc_flavor:
   docs_entry_points:
   - path: ./topics/markup/
     title: general guide
-  - path: ./topics/requirements/
-    title: guide to encoding requirements
+
+# This topic is not yet written
```

Comment:
We will need to write this. Do we have any content? At least we should make a new task for this.